examples/example_sdl_opengl3/Makefile

```diff
@@ -20,18 +20,32 @@ SOURCES += ../imgui_impl_sdl.cpp ../imgui_impl_opengl3.cpp
 SOURCES += ../../imgui.cpp ../../imgui_demo.cpp ../../imgui_draw.cpp ../../imgui_widgets.cpp
 OBJS = $(addsuffix .o, $(basename $(notdir $(SOURCES))))
 UNAME_S := $(shell uname -s)
+UNAME_M := $(shell uname -m)
 
 CXXFLAGS = -I../ -I../../
 CXXFLAGS += -g -Wall -Wformat
 LIBS =
 
+# There seems to be no way to detect Raspberry Pi from preprocessor defines
+# (i.e. gcc -dM -E - < /dev/null), so define something manually
+# See also https://www.raspberrypi.org/forums/viewtopic.php?p=200638
+ifeq ($(UNAME_M), armv7l)
+RPI = 1
+endif
+
 ##---------------------------------------------------------------------
 ## OPENGL LOADER
 ##---------------------------------------------------------------------
 
+ifndef RPI
```

Comment:
Currently if new platform flag was introduced this bit will have to be rewritten as it wont really make sense any more. Instead if we had `ifdef RPI` here we could just append a case of new platform in the future without altering old code.

Reply:
Sounds like a sane comment, looking into future.